app/src/main/java/ru/techpark/techparkhw/NumberFragment.java

```diff
@@ -0,0 +1,32 @@
+package ru.techpark.techparkhw;
+
+import android.graphics.Color;
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+public class NumberFragment extends Fragment {
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        View v = inflater.inflate(R.layout.big_number_fragment, container, false);
+        if (getArguments() == null) {
+            return v;
```

Comment:
I think it's good idea to show error message here or even throw the error that arguments must not be empty